easybuild/easyconfigs/p/PyTorch/PyTorch-1.11.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,116 @@
+    '10d1f0db8254f7564500fe97f134c3b4fa89156839b5f771a9396d80a0a1ada4',  # PyTorch-1.11.0_fix_skip_jit_cuda_fuser.patch
+    '9a04f4285b800dad8a00c3014af0a9713d40d5dd35d10931c7c0da4e89c558e9',  # PyTorch-1.11.0_fix_sharded_imports.patch
+    # PyTorch-1.11.0_increase_test_tolerances_TF32.patch
+    '26e179a4f6f57e49209092612ae5f5cd8c03fd2ca84566ba0244eabefc3736ba',
+    # PyTorch-1.11.0_increase_c10d_gloo_timeout.patch
+    '20cd4a8663f74ab326fdb032b926bf5c7e94d9750c515ab9050927ba00cf1953',
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('hypothesis', '6.13.1'),
+]
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Ninja', '1.10.2'),  # Required for JIT compilation of C++ extensions
+    ('Python', '3.9.5'),
+    ('protobuf', '3.17.3'),
+    ('protobuf-python', '3.17.3'),
+    ('pybind11', '2.6.2'),
+    ('SciPy-bundle', '2021.05'),
+    ('typing-extensions', '3.10.0.0'),
+    ('PyYAML', '5.4.1'),
+    ('MPFR', '4.1.0'),
+    ('GMP', '6.2.1'),
+    ('numactl', '2.0.14'),
+    ('FFmpeg', '4.3.2'),
+    ('Pillow', '8.2.0'),
+    ('cuDNN', '8.2.1.32', '-CUDA-%(cudaver)s', True),
+    ('magma', '2.6.1', '-CUDA-%(cudaver)s'),
+    ('NCCL', '2.10.3', '-CUDA-%(cudaver)s'),
+    ('expecttest', '0.1.3'),
+]
+
+# default CUDA compute capabilities to use (override via --cuda-compute-capabilities)
+cuda_compute_capabilities = ['3.5', '3.7', '5.2', '6.0', '6.1', '7.0', '7.2', '7.5', '8.0', '8.6']
+
+custom_opts = ["USE_CUPTI_SO=1"]
+
+excluded_tests = {
+    '': [
+        # Bad tests: https://github.com/pytorch/pytorch/issues/60260
+        'distributed/elastic/utils/distributed_test',
+        'distributed/elastic/multiprocessing/api_test',
+        # These tests fail on A10s at the very least, they time out forever no matter how long the timeout is.
+        # Possibly related to NCCL 2.8.3: https://docs.nvidia.com/deeplearning/nccl/release-notes/rel_2-8-3.html
+        # 'distributed/test_distributed_fork',
+        'distributed/test_distributed_spawn',
+        # Fails on A10s: https://github.com/pytorch/pytorch/issues/63079
+        'test_optim',
+        # Test from this suite timeout often. The process group backend is deprecated anyway
+        # 'distributed/rpc/test_process_group_agent',
+        # This test fails constently when run as part of the test suite, but succeeds when run interactively
+        'test_model_dump',
+    ]
+}
+
+runtest = 'cd test && PYTHONUNBUFFERED=1 %(python)s run_test.py --continue-through-error  --verbose %(excluded_tests)s'
+
+# The readelf sanity check command can be taken out once the TestRPATH test from 
+# https://github.com/pytorch/pytorch/pull/68912 is accepted, since it is then checked as part of the PyTorch test suite
+local_libcaffe2 = "$EBROOTPYTORCH/lib/python%%(pyshortver)s/site-packages/torch/lib/libcaffe2_nvrtc.%s" % SHLIB_EXT
+sanity_check_commands = [
+    "readelf -d %s | egrep 'RPATH|RUNPATH' | grep -v stubs" % local_libcaffe2,
```

Comment:
Yes, this now results in an error:
```
casparl@software2:~$ python -c 'import caffe2.python'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/casparl/.local/easybuild/Debian10/2021/software/PyTorch/1.11.0-foss-2021a-CUDA-11.3.1/lib/python3.9/site-packages/caffe2/python/__init__.py", line 2, in <module>
    from caffe2.proto import caffe2_pb2
  File "/home/casparl/.local/easybuild/Debian10/2021/software/PyTorch/1.11.0-foss-2021a-CUDA-11.3.1/lib/python3.9/site-packages/caffe2/proto/__init__.py", line 11, in <module>
    from caffe2.proto import caffe2_pb2, metanet_pb2, torch_pb2
ImportError: cannot import name 'metanet_pb2' from partially initialized module 'caffe2.proto' (most likely due to a circular import) (/home/casparl/.local/easybuild/Debian10/2021/software/PyTorch/1.11.0-foss-2021a-CUDA-11.3.1/lib/python3.9/site-packages/caffe2/proto/__init__.py)
```

and it seems this happens because the caffe2 framework has been disabled (see e.g. https://github.com/facebookresearch/d2go/issues/147#issuecomment-993149492 or any other of the many hits you get when searching `ImportError: cannot import name 'metanet_pb2' from partially initialized module 'caffe2.proto'`)

Reply:
OK, clear.